packages/react/src/primitives/Tabs/TabsContainer.tsx

```diff
@@ -20,6 +23,7 @@ const TabsContainerPrimitive: Primitive<TabsProps, 'div'> = (
   }: BaseTabsProps,
   ref
 ) => {
+  const groupId = useStableId();
```

Comment:
It would be helpful to add a small comment explaining the use case for `groupId` for future maintainers